src/ui/Select/Select.tsx

```diff
@@ -0,0 +1,164 @@
+  all: 'unset',
+  fontSize: 15,
+  lineHeight: 1,
+  color: '$text',
+  borderRadius: 3,
+  display: 'flex',
+  alignItems: 'center',
+  height: 36,
+  padding: '0 35px 0 25px',
+  position: 'relative',
+  userSelect: 'none',
+
+  '&[data-disabled]': {
+    color: '$surface',
+    pointerEvents: 'none',
+  },
+
+  '&:focus': {
+    backgroundColor: '$text',
+    color: '$white',
+  },
+});
+
+const StyledItemIndicator = styled(SelectPrimitive.ItemIndicator, {
+  position: 'absolute',
+  left: 0,
+  width: 25,
+  display: 'inline-flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+});
+
+const scrollButtonStyles = {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  height: 25,
+  backgroundColor: 'white',
+  color: '$grey10',
+  cursor: 'default',
+};
+
+const StyledScrollUpButton = styled(
+  SelectPrimitive.ScrollUpButton,
+  scrollButtonStyles
+);
+
+const StyledScrollDownButton = styled(
+  SelectPrimitive.ScrollDownButton,
+  scrollButtonStyles
+);
+
+// Exports
+export const RadixSelect = SelectPrimitive.Root;
+export const SelectTrigger = StyledTrigger;
+export const SelectValue = SelectPrimitive.Value;
+export const SelectIcon = SelectPrimitive.Icon;
+export const SelectContent = StyledContent;
+export const SelectViewport = StyledViewport;
+export const SelectGroup = SelectPrimitive.Group;
+export const SelectItem = StyledItem;
+export const SelectItemText = SelectPrimitive.ItemText;
+export const SelectItemIndicator = StyledItemIndicator;
+export const SelectScrollUpButton = StyledScrollUpButton;
+export const SelectScrollDownButton = StyledScrollDownButton;
+
+export type SelectOption = {
+  value: React.ReactText;
+  label: React.ReactText;
+};
+
+export const Select = (
+  props: SelectProps & {
+    options: SelectOption[];
+  }
+) => {
+  const { defaultValue, options } = props;
+
+  return (
+    <RadixSelect defaultValue={defaultValue} {...props}>
+      <SelectTrigger>
+        <SelectValue />
+        <SelectIcon>
+          <ChevronDownIcon color="#b8bdbf" />
+        </SelectIcon>
+      </SelectTrigger>
+      <SelectContent>
+        <SelectScrollUpButton>
+          <ChevronUpIcon color="#b8bdbf" />
+        </SelectScrollUpButton>
+        <SelectViewport>
+          <SelectGroup>
+            {options.map(({ value, label }) => (
+              <SelectItem value={String(value)} key={value}>
+                <SelectItemText>{label}</SelectItemText>
+                <SelectItemIndicator>
+                  <CheckIcon />
+                </SelectItemIndicator>
+              </SelectItem>
+            ))}
+          </SelectGroup>
+        </SelectViewport>
+        <SelectScrollDownButton>
+          <ChevronDownIcon />
+        </SelectScrollDownButton>
+      </SelectContent>
+    </RadixSelect>
+  );
+};
+
+/* Storybook utility for stitches variant props
+
+NOTE: this can't live in the stories file because the storybook navigator will take a story and will crash
```

Comment:
hmm, can you say more about the error that was happening here? did we document this issue or talk about it somewhere else?

Reply:
@crabsinger this is an older version of the code, unsure how you're seeing this still

Reply:
It wasn't added by me and it's in most stories, just followed the pattern. The commit that introduced that was https://github.com/karelianpie/coordinape/commit/258a2bcd1cd042aeb57027af6680361c25f741e5 but it's in a few components `src/ui/Button/Button.tsx`, `src/ui/IconButton/IconButton.tsx`, etc.

Reply:
👍